src/python/dxpy/scripts/dx.py

```diff
@@ -4646,7 +4646,7 @@ def positive_integer(value):
 parser_make_download_url = subparsers.add_parser('make_download_url', help='Create a file download link for sharing',
                                                  description='Creates a pre-authenticated link that can be used to download a file without logging in.',
                                                  prog='dx make_download_url')
-path_action = parser_make_download_url.add_argument('path', help='Data object ID or name to access')
+path_action = parser_make_download_url.add_argument('path', help='Project-qualified data object ID or name, e.g. project-xxxx:file-yyyy, or project-xxxx:/path/to/file.txt')
```

Comment:
@commandlinegirl I've just removed trailing dot to make it consistent with other commands and help texts. Is it ok?

Reply:
LG, thanks!